docs/VIDEO_RECORDING_AND_TRAINING_WORKFLOW.md

```
@@ -244,183 +234,63 @@ Content-Disposition: attachment; filename="amy_model.npz"
+export async function fetchMlpModelWithFallback(params: {
+  endpoint: string;
+  token?: string;
+  profileId?: string;
+}): Promise<MlpModelResponse | null>
 ```
 
 ---
 
 ## Complete Workflow Diagram
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)